### ISSUE TITLE
Lua 5.3 support

### ISSUE THREAD

**Issue by daurnimator** [deepakjois/luaharfbuzz#36]
```
 $ sudo luarocks install luaharfbuzz
Warning: falling back to curl - install luasec to get native HTTPS support

Error: luaharfbuzz supports only Lua 5.2 but not Lua 5.3.
```

I have not looked any further for now.

**Comment by deepakjois** [comment]
I suppose the problem is with this line in the Rockspec:

```
"lua ~> 5.2"
```

Might need to change it to:

```
"lua" >= 5.2
```

Let me try if it works and fix.

**Comment by deepakjois** [comment]
Fixed in a3ce366835feb0e722866b25c39c5942bdecd422
